portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -31,6 +31,21 @@
 import com.google.appengine.api.datastore.Query.SortDirection;
 import com.google.appengine.api.users.UserService;
 import com.google.appengine.api.users.UserServiceFactory;
+// Imports for uploading images.
```

Comment:
nit: typically you do not add comments for imports.

Reply:
got it!

done